wdl-lint/DEFINITIONS.md

```diff
@@ -0,0 +1,46 @@
+# Definitions
+
+This document defines key concepts and terminology used throughout the WDL linting rules.
+
+## WDL Document Structure
+
+### Preamble
+
+The document preamble is defined as anything before the version declaration statement and the version declaration statement itself. Only comments and whitespace are permitted before the version declaration.
+
+## Comment Types
+
+### Lint Directives
+
+Lint directives are special comments that begin with `#@ except:` followed by a comma-delimited list of rule IDs. These comments are used to disable specific lint rules for a section of the document. When a lint directive is encountered in the preamble, it will disable the specified rules for the entire document. For example:
+
+```wdl
+#@ except: LineWidth, CommentWhitespace
+```
+
+Lint directives in the preamble should not have any whitespace before the comment. If lint directives are present in the preamble, they should be at the absolute beginning of the document. Multiple lint directives are permitted, but they should not be interleaved with preamble comments or blank lines.
+
+Lint directives elsewhere in the file should match the indentation of whatever they are "commenting on".
```

Comment:
```suggestion
```
same here. I think we can leave the details of formatting up to the lint rules themselves.